vhdbuilder/scripts/automate_generate_version.sh

```diff
@@ -4,22 +4,23 @@ set -euxo pipefail
 build_ids=$1
 global_image_version="${IMAGE_VERSION:=}"
 for build_id in $build_ids; do
-    for artifact in $(az pipelines runs artifact list --run-id $build_id | jq -r '.[].name'); do    # Retrieve what artifacts were published
+    artifacts=($(az pipelines runs artifact list --run-id $build_id | jq -r '.[].name'))
+    artifacts=(${artifacts[@]//vhd*})
```

Comment:
this filters out all the files starting with vhd, so all that is left in the artifacts list is publishing info, which is what we really ever needed.